src/main/java/net/pod/peaengine/physics/Vector2D.java

```diff
@@ -0,0 +1,18 @@
+package net.pod.peaengine.physics;
+
+import java.util.Vector;
+
+public class Vector2D {
+    public double x;
+    public double y;
+
+    public Vector2D(double x, double y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + x + ", " + y + "]";
```

Comment:
well im being a bit nitpicky here, but for vectors and positions we use round brackets. Square bracckets are for collections.